source/cloud/cloud-integrations/cloud-bot-accounts.rst

```diff
@@ -53,7 +53,7 @@ User Interface (UI)
 ^^^^^^^^^^^^^^^^^^^
 
 1. Go to **Main Menu > Integrations > Bot Accounts**.
-2. Click **Add Bot Account**.
+2. Select **Add Bot Account**.
 3. Set the **Username** of the bot. The username must begin with a letter, and contain between 3 and 22 lowercase characters made up of numbers, letters, and the symbols ".", "-", and "_".
```

Comment:
Does it have to contain all of those or can it be just letters or just letters and underscores etc?

Reply:
Great question, @justinegeffen! Who do you recommend can technically confirm?

Reply:
Realted to below - perhaps we make the point here that bot usernames should be unique and not conflict with any existing user accounts.